pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1312,7 +1312,7 @@ const (
 	DefTiDBEnable1PC                               = false
 	DefTiDBGuaranteeLinearizability                = true
 	DefTiDBAnalyzeVersion                          = 2
-	DefTiDBAutoAnalyzePartitionBatchSize           = 128
+	DefTiDBAutoAnalyzePartitionBatchSize           = mysql.PartitionCountLimit
```

Comment:
Please add the comment to mark it as depreciated.

Reply:
I have added comments.